.github/workflows/quay.yml

```diff
@@ -0,0 +1,16 @@
+name: quay
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We may want to just have this trigger on pull_requests/push at the moment, so we don't have to force merge to get it working initially.

Reply:
Ah just for testing you mean? Then put it back when we merge?

Reply:
Yeah, exactly. We can revert back once we try and play around with it a bit more.